cypress/integration/plugins/alerting-dashboards-plugin/composite_level_monitor_spec.js

```diff
@@ -71,26 +71,30 @@ describe('CompositeLevelMonitor', () => {
       cy.get('[data-test-subj="visualEditorRadioCard"]').click({ force: true });
 
       // Wait for input to load and then type in the monitor name
-      cy.get('input[name="name"]').type(SAMPLE_VISUAL_EDITOR_MONITOR);
+      cy.get('input[name="name"]').type(SAMPLE_VISUAL_EDITOR_MONITOR, {
+        force: true,
+      });
 
       // Select associated monitors
       cy.get('[data-test-subj="monitors_list_0"]').type('monitorOne', {
         delay: 50,
+        force: true,
       });
       cy.get('[title="monitorOne"]').click({ force: true });
 
       cy.get('[data-test-subj="monitors_list_1"]').type('monitorTwo', {
         delay: 50,
+        force: true,
       });
       cy.get('[title="monitorTwo"]').click({ force: true });
 
       cy.get('button').contains('Add trigger').click({ force: true });
 
       // Type trigger name
       cy.get('[data-test-subj="composite-trigger-name"]')
-        .type('{selectall}')
-        .type('{backspace}')
-        .type('Composite trigger');
+        .type('{selectall}', { force: true })
```

Comment:
in the past I saw potentially cypress's focus are taken over, so would the typing delay here help as well (like on line 86)

Reply:
There we have delay because it is a combo-box and we wait for the dropdown to show up, where as here it is an input box, so we should not need a delay

Reply:
Though I will take a second pass later to ensure we have focus calls on all the input elements to make the tests more robust